meson.build

```
@@ -45,8 +45,10 @@ else
 endif
 
 if os != 'Linux'
+  cap_dep = []
   kvm_dep = cc.find_library('kvm', required: true)
 else
+  cap_dep = dependency('libcap', version: '>=2.33', required : true)
```

meson's default for required is already true, we don't need to specify it:
```suggestion
  cap_dep = dependency('libcap', version: '>=2.33')
```